int16 left
int16 right
---
string message
